build.sbt

```diff
@@ -204,6 +204,7 @@ lazy val core = project
     name := "akka-stream-kafka",
     AutomaticModuleName.settings("akka.stream.alpakka.kafka"),
     libraryDependencies ++= Seq(
+        "com.typesafe.akka" %% "akka-actor-typed" % akkaVersion,
```

Comment:
Hm, I'm not sure we want to add the dependency.
My original thought was to show Akka Typed in the snippets with the necessary imports for conversions where needed.

@patriknw Opinions?

Reply:
I agree, it's not worth adding the dependency just for this (rebalance listener).

Reply:
Ok, I'll revert the changes to core and move the typed to classic interoperability to the docs example.

Reply:
thanks